src/components/Header/Header.tsx

```diff
@@ -421,13 +434,14 @@ const Header: React.FC = () => {
       )}
       <Box>
         <Box
-          width={36}
+          width={45}
           height={36}
           display='flex'
           alignItems='center'
           justifyContent='center'
           marginRight={1}
         >
+          <StyledPollingDot></StyledPollingDot>
```

Comment:
what is this?

Reply:
This is the green dot added by Sameep that shows the interface is secure